core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -71,64 +72,129 @@ object ConsumerGroupCommand {
       }
     } catch {
       case e: Throwable =>
-        println("Error while executing consumer group command " + e.getMessage)
-        println(Utils.stackTrace(e))
+        consumerGroupOutputWriter.printError("Executing consumer group command failed due to " + e.getMessage
+            + "\n" + Utils.stackTrace(e))
     } finally {
       consumerGroupService.close()
     }
   }
 
+  case class ConsumerGroupAssignment(group: String, topicOpt: Option[String], partitionOpt: Option[Int], offsetOpt: Option[Long],
+                                     lagOpt: Option[Long], memberIdOpt: Option[String], memberHostOpt: Option[String],
+                                     clientIdOpt: Option[String], logEndOffset: Option[Long])
+
+  trait OutputWriter {
+    def printError(msg: String)
+    def printConfirmation(msg: String)
+    def printAssignmentHeader()
```

Comment:
That's fair. Would it make sense to use `Assignments`/`assignments` to imply all assignments, and `Assignment`/`assignment` to imply a single assignment row?